anta/cli/nrfu/utils.py

```diff
@@ -90,8 +90,10 @@ def print_table(ctx: click.Context, group_by: Literal["device", "test"] | None =
         console.print(reporter.report_summary_devices(results))
     elif group_by == "test":
         console.print(reporter.report_summary_tests(results))
+    elif ctx.obj["expand_atomic"]:
+        console.print(reporter.report_expanded(results))
     else:
-        console.print(reporter.report_all(results))
+        console.print(reporter.report(results))
```

Comment:
Add this logic to other functions as well if we want to support it.

Reply:
I think it just makes sense to provide an option for Table and Text.
Other reports needs the atomic results anyway.